harvester_e2e_tests/integrations/test_1_volumes.py

```diff
@@ -30,6 +41,42 @@ def ubuntu_image(api_client, unique_name, image_ubuntu, polling_for):
                     api_client.images.get, image_name)
 
 
+@pytest.fixture(scope="module")
+def bad_ubuntu_image(api_client, unique_name, image_ubuntu, polling_for):
+    """
+    Generates a Ubuntu image with a bad sha512 checksum
+
+    1. Creates an image name based on unique_name
+    2. Create the image based on URL with a bad statically assigned checksum
+    3. Response for creation should be 201
+    4. Loop while waiting for image to be created
+    5. Yield the image with the namespace and name
+    6. Delete the image
+    7. The response for getting the image name should be 404 after deletion
+    """
+
+    image_name = f"img-{unique_name + '-badchecksum'}"
+    # Random fake checksum to use in test
+    fake_checksum = "241c43e5b63fe2fd8a43a772469f03015\
```

Comment:
we can use something like:
https://github.com/harvester/tests/blob/958ff032ea48bc9a59d213ebdc7cbf09eac53c5b/harvester_e2e_tests/integrations/test_upgrade.py#L393

to generate checksum.